lldb/packages/Python/lldbsuite/test/dotest.py

```diff
@@ -428,6 +428,9 @@ def parseOptionsAndInitTestdirs():
         configuration.clang_module_cache_dir = os.path.join(
             configuration.test_build_dir, 'module-cache-clang')
 
+    if args.lldb_swift_libs:
+        configuration.lldb_swift_libs = args.lldb_swift_libs
+
```

Comment:
Should this also be in auto-populated in https://github.com/apple/llvm-project/blob/next/lldb/utils/lldb-dotest/lldb-dotest.in ?